### DISCUSSION TITLE
No way to link to a metric?

### DISCUSSION THREAD

**Discussion by Cyberax** [uptrace/uptrace#450] [General]
Is there a way to link to a metric in the "Explore Metrics" view? I don't think there's any:

<img width="1553" alt="изображение" src="https://github.com/user-attachments/assets/dd69ba90-a482-424b-bf0a-d4179483033c" />

So it's not possible to send these views over email/slack/...

**Comment by vmihailenco** [comment]
No, that's just a dialog and you can't link to it. For now, I suggest to create a dashboard if you want to share anything.

**Reply by Cyberax** [reply]
Thank you! I assume that there's a plan to change this? It's really painful to create one-off dashboards when using metrics for incident investigation.
